screens/ChatsScreen.js

```diff
@@ -146,17 +107,17 @@ class ChatsScreen extends React.Component {
     }
 }
 
-function mapStateToProps({ chats }) {
-    return { chats };
-}
-
-const mapDispatchToProps = {
-    createMessage,
-    setCurrentUser,
-    setCurrentChat,
-    createChat
+const mapStateToProps = ({ chatsReducer: { chats } }) => {
+    return {
+        chats
+    };
 };
 
+const mapDispatchToProps = (dispatch) => ({
+    fetchChats: bindActionCreators(fetchChats, dispatch),
```

Comment:
important - need to follow this process of bindActionCreators for any calls to actions in components.